modules/image-integrations-section.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="image-integrations-section_{context}"]
+= Image Integrations section
+
+The *Image Integrations* section organizes the information into the following groups:
+
+Name:: Indicates the name of the integrated image registry.
```

Comment:
Image integrations can also represent a scanner, for example

<img width="1389" alt="image" src="https://github.com/user-attachments/assets/097a02cc-d644-4770-984c-79810ecc0a17" />

Recommend removing the reference to `registry`